agent/util-scripts/pbench-tool-meister-stop

```diff
@@ -38,22 +42,42 @@ def is_running(pid):
     return True
 
 
-def main(argv):
-    """Main program for the tool meister stop.
+def main(_prog, cli_params):
+    """Main program for the tool meister stop CLI interface.
+
+    Stopping the Tool Meisters involves four steps:
+
+    1. End ("end") the run of the persistent tools
+
+    2. Collect any requested system configuration information ("sysinfo")
+
+    3. Record in the local metadata.log file all relevant metadata from the
+    stop operation, including the timing to end the persistent tools, the
+    timing to collect the system information, and any other data regarding the
+    outstanding state of the Tool Meisters.
+
+    4. Sends the "terminate" message to the redis server so that all connected
+    services, tool-meisters, tool-data-sink, etc. will shutdown.  Once all
+    services acknowledge the receipt of the "terminate" message, we look to
+    ensure that the local Tool Data Sink and optional Tool Meister instance
```

Comment:
... if we created the local Tool Data Sink and optional Tool Meister ...

Reply:
Actually, this comment really applies to PR #2076, as in this PR we always create the local Tool Data Sink and optional local Tool Meister.